iocage_lib/ioc_start.py

```diff
@@ -275,18 +275,13 @@ def __start_jail__(self):
             net = []
 
             if ip4_addr != "none":
-                gws = netifaces.gateways()
+                ip4_addrs = []
 
                 for _ip4_addr in ip4_addr.split(","):
-                    if "|" not in _ip4_addr:
-                        try:
-                            def_iface = gws["default"][netifaces.AF_INET][1]
-                            _ip4_addr = f'{def_iface}|{_ip4_addr}'
-                        except KeyError:
-                            # Best effort for default interface
-                            pass
+                    ip4_addrs.append(f"{_ip4_addr}")
 
-                    net.append(f"ip4.addr={_ip4_addr}")
+                ip4_addrs_str = ",".join(ip4_addrs)
+                net.append(f"ip4.addr={ip4_addrs_str}")
```

Comment:
I think these lines can be squashed down to a single line, as ip4_addrs isn't used anywhere else

```
net.append(f'ip4.addr={ip4_addr}')
```

Reply:
Will put that in with the next changes.